src/test/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RecommendationServiceTest.kt

```diff
@@ -112,8 +113,12 @@ internal class RecommendationServiceTest : ServiceTestBase() {
       given(recommendationRepository.save(any())).willReturn(recommendationToSave)
       recommendationService = RecommendationService(recommendationRepository, mockPersonDetailService, templateReplacementService, userAccessValidator, convictionService, riskServiceMocked, communityApiClient, mrdEmitterMocked)
 
+      // and
+      val featureFlags = if (recallConsidered == Status.RECALL_CONSIDERED.toString()) FeatureFlags(flagConsiderRecall = true) else null
```

Comment:
Guess this is fine, just not 100% sure if Status.RECALL_CONSIDERED is the correct thing to be doing the equals to here. You're testing the feature flag value which doesn't really have anything to do with the `Status` object. If you want to leave this in, we just need to remember to clean this test up when we come to removing the feature flag in a few weeks time, as it might not be obvious.